### ISSUE TITLE
Modifiy 'tests/org/tests_edit_org_api.py' and add 'super(EditOrgTestCase,self).tearDown()' function at the end.

### ISSUE THREAD

**Issue by sahil9001** [ECellNitrr/teesco-backend#101]
**Is your feature request related to a problem? Please describe.**
Currently the ```def tearDown(self):``` function only deletes the ```auth_user``` after each test but we want to delete whole objects created by ```setUp(self) and super class setUp()``` .

**Describe the solution you'd like**
Call ```super(EditOrgTestCase,self).tearDown()```  in tests/org/tests_edit_org_api.py in tearDown(). Also make other appropriate changes to tearDown() as needed.

**Comment by adityatiwari23** [comment]
I am not very much sure that I will be able to fix the issue easily but I want to have a try.
If you could guide how to resolve the issue, it would be very helpful for me.

**Comment by sahil9001** [comment]
https://github.com/ECellNitrr/teesco-backend/blob/e31ee99aa6fbbba3309d3c71a437556eb15841f1/tests/org/tests_edit_org_api.py#L76

This above function call deletes the `auth_user` only, but we want to delete all of the objects created by `test database` after each `test`, so you could replace that line with `super(EditOrgTestCase,self).tearDown()`

**Comment by adityatiwari23** [comment]
@sahil9001 sir
Should i just replace  ~~`self.auth_user.delete()`~~  with 
`super(EditOrgTestCase,self).tearDown()`

Or this should be added at the last

**Comment by sahil9001** [comment]
Yes, that's what you want to do here. Just replace `self.auth_user.delete()` with `super(EditOrgTestCase,self).tearDown()`

**Comment by priyanshisharma** [comment]
@sahil9001 @Adityatiwar1 
# Sorry :sweat:  Good first issues are reserved for FIRST TIME CONTRIBUTORS
Since, @Adityatiwar1 has already contributed to this repository by solving #113 with #126 , he isn't eligible to work on this. Sorry to let you down.
